generic_throttle/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(generic_throttle)
+
+find_package(catkin REQUIRED COMPONENTS
+  rospy
+  roslib
+  std_msgs
+  sensor_msgs
+  visualization_msgs
```

Comment:
entries only needed for c++ code...not for python....remove the ROS packages...just keep catkin...